packages/amplication-client/src/Purchase/FAQ.tsx

```diff
@@ -74,23 +93,37 @@ export const FAQ = () => {
         />
         <Question
           question="What if I need to add more team members?"
-          answer="No problem. Select the plan that meets your needs. The Free edition supports up to two user seats, the Pro edition supports up to ten user seats, and the Enterprise edition has no limit. "
+          answer="No problem. Select the plan that meets your needs. The Free edition supports up to two user seats, the Pro edition supports up to five user seats, and the Enterprise edition has no limit. "
         />
         <Question
           question="Can I pay for my subscription with an invoice?"
-          answer={`Companies can use invoices to purchase Enterprise subscriptions. Please <contact sales> for more information and to purchase.
-
-          link is to send an email to sales@amplication.com
-          
-          email subject - “Amplication Enterprise Plan Inquiry”`}
+          answer={
+            <div>
+              Companies can use invoices to purchase Enterprise subscriptions.
+              Please{" "}
+              <MailTo
+                email="mailto:sales@amplication.com"
+                subject="Amplication Enterprise Plan Inquiry"
+                body="contact sales"
+              />{" "}
```

Comment:
it looks like you are missing a few words before the email link
please add a screenshot of the fixed UI

Reply:
<img width="1286" alt="image" src="https://github.com/amplication/amplication/assets/97830649/94e0cf5e-675e-4d86-a629-f0002dd495cf">
@yuval-hazaz  looks good to me. Can you please approve?